### PR TITLE
feat: add custom logs to openvpn

### PR DESCRIPTION
**Pull request by polxpolx**
Added the custom template for openvpnas

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -27,3 +27,4 @@ packages:
   - google-fluentd
   - google-fluentd-catch-all-config
 stackdriver_collectd_service: stackdriver-agent
+stackdriver_openvas: False
```
Comment (ahached) [inline defaults/main.yml]: Please change False to false

**Review by ahached — CHANGES_REQUESTED** [review]
Please explain the variable usage in the README.md, if I understood you want to enable a custom config for stackdriver only for openvpn server. If we want to apply this change we will also need to create a PR on the techops-tool to override the default  stackdriver_openvas in openvpn playbook

File: defaults/main.yml (hunk 1 above)
Comment (polxpolx) [reply defaults/main.yml]: Done added information inside the readme. If you merge inside the dedicated branch v6 than we will have nothing to change inside the techops repo. Please merge the PR.

**Review by ahached — APPROVED** [review]
LGTM

**Pull request merged**